src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -802,7 +912,7 @@ int tap_esc_start(void)
 
 	if (tap_esc == nullptr) {
 
-		tap_esc = new TAP_ESC;
+		tap_esc = new TAP_ESC(_supported_channel_count);
```

Comment:
Refactor and pass param using argv and then to constructor. I am surprised the uart woks opened on caller thread and passed to new task.